shared/chat/create-channel/index.desktop.js

```diff
@@ -8,15 +8,16 @@ import type {Props} from '.'
 const CreateChannel = (props: Props) => (
   <PopupDialog onClose={props.onClose} styleCover={_styleCover} styleContainer={_styleContainer}>
     <Box style={_boxStyle}>
-      <Avatar isTeam={true} teamname={props.teamname} size={16} />
+      <Avatar isTeam={true} teamname={props.teamname} size={24} style={{minHeight: 24}} />
```

Comment:
i'm surprised the minHeight is necessary. likely should be enforced in avatar itself

Reply:
Ok - moved to avatar-render